workers/unity/Packages/com.improbable.gdk.core/GameObjectRepresentation/ComponentIdAttribute.cs

```diff
@@ -0,0 +1,21 @@
+﻿using System;
+
+namespace Improbable.Gdk.Core
```

Comment:
Shouldn't all of this be in a feature module rather than the Core?

Reply:
Our decision for now is to put the monobehaviour workflow into core. We can still separate it out later.

Reply:
We will almost certainly move this out as nothing needs to be in core.
I actually think we should revisit the decision to have anything in core.

Reply:
If we want to move everything out of core then that should be done in a separate PR at a different time. The decision in the meeting was made just to prevent deadlocks like this.

Reply:
We can do it easily just before merging the final state of the feature branch into master. Right now there's monobehaviour things in different PRs / repos to-be-made-prs all in the Core namespace, changing that would be a hassle

Reply:
I would prefer to do it earlier, but agree that it doesn't have to be done in this pr